config.md

```diff
@@ -69,7 +69,9 @@ e.g. `PathMatch: [foo/.*, bar/.*]` matches files in either directory.
 
 Conditions based on a file's path use the following form:
 
-- if the fragment came from a project directory, the path is relative
+- if the fragment came from a project directory, the path is relative  
+(note: `./dir/.*` is an incorrect form for `dir` content in the project's root
```

Comment:
I have trouble understanding this sentence (not sure what the "regex perspective" refers to). And listing incorrect examples can add extra confusion. Could this be:

> (e.g. for `/proj/.clangd`, the file `/proj/x/y.cc` has a path of `x/y.cc`)

Reply:
Looks like I misunderstood @HighCommander4 explanation.
It's a common practice to start a path with `./` talking about relative paths. But the config parser doesn't understand that form, it requires starting away with the name, right?
"regex perspective" is my wrong explanation because regex with `.` special character can be any character including `.`, so according to my logic, it should have worked. Forget it.
Yeah, your example looks better in every sense.